markup.go

```diff
@@ -154,26 +150,49 @@ func (m ClassMap) Apply(h *HTML) {
 	Property("className", strings.Join(classes, " ")).Apply(h)
 }
 
-// List represents a list of Markup, Component, or HTML which is individually
+// MarkupList represents a list of Applyer which is individually
 // applied to an HTML element or text node.
-type List []MarkupOrComponentOrHTML
+type MarkupList []Applyer
 
-// Apply implements the Markup interface.
-func (l List) Apply(h *HTML) {
-	for _, m := range l {
-		apply(m, h)
+// Apply implements the Applyer interface.
+func (m MarkupList) Apply(h *HTML) {
```

Comment:
I personally wonder if having this exported makes sense or not.

I think the number of situations where someone will use it is quite low, and they could use `var markup []vecty.Applyer` & `vecty.Markup(markup...)` for such cases.

My instinct says this should be unexported until we have good evidence that it would be fairly useful to people, this way we keep our exported API footprint smaller.

Reply:
Yep, this was left exposed by accident.